config/initializers/20_paperclip.rb

```diff
@@ -1,15 +1,28 @@
-if AWSConfig.config.aws_access_id.blank? && AWSConfig.config.aws_secret_key.blank?
-  cred = {
-    bucket: AWSConfig.config.image_bucket_name
-  }
-else
-  cred = {
-    bucket: AWSConfig.config.image_bucket_name,
-    access_key_id: AWSConfig.config.aws_access_id,
-    secret_access_key: AWSConfig.config.aws_secret_key,
+if CloudConfig::service_name == 'aws'
+  if AWSConfig.config.aws_access_id.blank? && AWSConfig.config.aws_secret_key.blank?
+    cred = {
+      bucket: AWSConfig.config.image_bucket_name
```

Comment:
__[rubocop]__
* Style/TrailingCommaInLiteral: Put a comma after the last item of a multiline hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)